package.json

```
@@ -20,14 +20,15 @@
   "scripts": {
     "test:node": "mocha test/test-*.spec.js",
     "test:browser": "playwright-test test/test-*.js",
-    "test:react-native": "jest test/test-lib.react-native.spec.js",
+    "test:react-native": "npm test --prefix test/react-native",
+    "test:create-react-app": "npm test --prefix test/create-react-app",
```

same with the above suggestion.

```suggestion
    "test:create-react-app": "cd test/create-react-app && npm test",
```